GVRf/Framework/framework/src/main/jni/objects/scene_object.cpp

```diff
@@ -467,7 +473,7 @@ bool SceneObject::intersectsBoundingVolume(SceneObject *scene_object) {
 
 
 BoundingVolume& SceneObject::getBoundingVolume() {
-    if (!bounding_volume_dirty_ && !transform_dirty_) {
```

Comment:
@NolaDonato Any memory of why you had to add the check in first place? How to verify that whatever you fixed is still working correctly?

```
commit 9f3f152be4d9e4a9b37beb2e9da5a57673f33f8a
Author: Nola Donato <nola.donato@samsung.com>
Date:   Mon Jan 23 23:50:53 2017 -0800

    fixed problem with bounding box computation
    
    which caused the cursor to be incorrectly culled in some of the demos
```